src/org/zaproxy/zap/extension/pscanrules/resources/help/contents/pscanrules.html

```diff
@@ -59,11 +59,16 @@
 At HIGH threshold this scanner does not alert on client or server error responses, for all other enabled 
 thresholds it alerts on all response types if the header is missing or set to something other than 'nosniff'.
  
-<H2>X-Frame-Option</H2>
+<H2>X-Frame-Options Header Scanner</H2>
 This scanner checks for the existence and validity of the X-FRAME-OPTIONS header. At HIGH threshold this scanner
```

Comment:
For consistency the "X-FRAME-OPTIONS" could be changed to match the case of other entries.